thirdparty/README.md

```diff
@@ -393,6 +393,23 @@ Files extracted from the upstream source:
 - All .h files in `src/`
 - LICENSE.txt
 
+##open-simplex-noise
```

Comment:
There should be a space after `##`, and two lines between sections.

That being said, since it's only two files, I think it would be better to just put them in `thirdparty/misc` (and document them in the relevant section). You can likely drop the README and maybe name the license file `open-simplex-noise-LICENSE`.